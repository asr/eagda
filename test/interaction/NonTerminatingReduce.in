top_command (cmd_load currentFile [])

-- Should be '0'
top_command (cmd_compute_toplevel False "loop 5")

-- Should be 'loop 5'
goal_command 0 (cmd_compute False) "loop 5"

