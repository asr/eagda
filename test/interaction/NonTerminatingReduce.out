Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking NonTerminatingReduce (NonTerminatingReduce.agda). " t)
(agda2-info-action "*Type-checking*" "Finished NonTerminatingReduce. " t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set " nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> Agda2> Agda2> (agda2-status-action "")
(agda2-info-action "*Normal Form*" "0" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> Agda2> Agda2> (agda2-status-action "")
(agda2-info-action "*Normal Form*" "loop 5" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> Agda2> 